doc/contributing/coding-conventions.chapter.md

```diff
@@ -643,7 +643,7 @@ Here are examples of package tests:
 
 Like [package tests](#ssec-package-tests-writing) as shown above, [NixOS module tests](https://nixos.org/manual/nixos/stable/#sec-nixos-tests) can also be linked to a package, so that the tests can be easily run when changing the related package.
 
-For example, assuming we're packaging `nginx`, we can link its module test via `passthru.tests`:
+The basic way to set the `tests` attribute, if we're willing to make some assumptions, is the following. We use the `nginx` package as an example:
```

Comment:
```suggestion
The basic way to set the `tests` attribute is the following.
We use the `nginx` package as an example:
```

Inserting that sentence makes the whole thing hard to parse. Also, you clarify the assumptions after the example, so it should be fine.